modules/oe_editorial_entity_version/src/Form/CorporateWorkflowEntityRevisionRevertForm.php

```diff
@@ -127,46 +128,42 @@ public function getConfirmText() {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, $node_revision = NULL) {
-    // We need to anticipate string or NodeInterface types of $node_revision to
-    // be compatible with both core versions v9.2 and v9.3.
-    // @see: https://www.drupal.org/project/drupal/issues/2730631
-    $this->revision = $node_revision;
-    if (!$this->revision instanceof NodeInterface) {
-      $this->revision = $this->entityTypeManager->getStorage('node')->loadRevision($node_revision);
-    }
+  public function buildForm(array $form, FormStateInterface $form_state, $_entity_revision = NULL) {
```

Comment:
$entity_revision

Reply:
No, this is the param name that is being passed.